### PR TITLE
Upgrade version number to 1.1.2

### PR DESCRIPTION
**Pull request by wenyongh**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -12,6 +12,111 @@
...
+Enable Nuttx spec test option and register aot symbols
+Use wabt binary instead of building from source in spec test
+Fix sample ref-types/wasm-c-api build error with wat2wasm low version
+nuttx: Enable ref types by Kconfig
+Update xtensa LLVM version to 15.x
+Add bh_print_proc_mem() to dump memory info of current process
+Create trap for error message when wasm_instance_new fails
+wamr-test-suites: Add support for ARM/RISCV by QEMU
+Fix zephyr sample build errors
+Fix source debugger error handling: continue executing when detached
```
Comment (loganek) [inline RELEASE_NOTES.md:81]: I think this is more like a bugfix?

**Comment by loganek** [comment]
Great to see so many great changes since the last version! :confetti_ball:

File: RELEASE_NOTES.md (hunk 1 above)
Comment (wenyongh) [reply RELEASE_NOTES.md]: Yes, sometimes it is not easy to classify whether it is bugfix or enhancement. I moved several items to "Bug Fixes".

**Comment by wenyongh** [comment]
> Great to see so many great changes since the last version! 🎊

Yes, it is amazing. We will do more test and release the new version if all goes well.

**Pull request merged**